docs/packages-icons.md

```diff
@@ -1,11 +1,30 @@
 # Icons package
 
-This package bundles all project-specific svg icons in a node module.
+This package bundles all project-specific SVG icons in a node module.
 
-It uses the `svgr` CLI and typescript to turn a source set of svg icons into a resultant sent of
-react components. The steps are:
+It uses the `svgr` CLI and typescript to turn a source set of SVG icons into a resultant set of
+React components. The steps are:
 
 1. Use `svgr` to convert icons in `src/main/svg` to tsx components in `src/main/generated`.
-
-2. Run a typescript build to convert the `generated` components into ES2020 modules in the `dist`
+1. Run a typescript build to convert the `generated` components into ES2020 modules in the `dist`
    folder.
+
+## Contribution
+
+To add a new icon that isn't available as a
+[Carbon Icon](https://carbondesignsystem.com/guidelines/icons/library/) or
+[Carbon Pictogram](https://carbondesignsystem.com/guidelines/pictograms/library/):
+
+1. Download the latest [Sketch file from Box](https://ibm.ent.box.com/folder/158366227035).
+1. Add attribution where required so we can do manual open-source audits for all icons that didn't
+   come from Carbon icon and pictogram code packages.
+1. Size the SVG at its intended size so we can be pixel-precise for standard density displays (SVGs
+   can get resized in the web app so this isn't overly important.)
+1. Include an internal 2px padding for 32px icons per
+   [IBM Design Language specifications](https://www.ibm.com/design/language/iconography/ui-icons/design/#padding)
+   (1px padding for 16px icons.)
+1. Ensure the fill colors are all black.
```

Comment:
Are we not storing icons with colors in this package?
See https://next.carbondesignsystem.com/developing for example
![Screen Shot 2022-06-16 at 10 07 51 AM](https://user-images.githubusercontent.com/2753488/174101289-1335fd49-6043-4dc9-8c79-1eb3324b3e7c.png)

Reply:
Right now we just have black SVGs so they can take a single color (e.g. in catalog items.)

If we need colored SVGs that need multiple colors, we'll need to add those as separate SVGs.

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/1691245/174102522-be4462f2-5cd8-441e-917f-89817ed94ca3.png">

Reply:
@alisonjoseph oh you're talking about all these? Maybe later we can do cleanup so these other icons (that got ported over from carbon-website) are in our Sketch source and properly attributed like the others.

<img width="306" alt="image" src="https://user-images.githubusercontent.com/1691245/174121798-e7c5ec13-9192-40b9-bd9a-27fcb36ec2c1.png">